cmd/cluster-kube-controller-manager-operator/render/render.go

```diff
@@ -20,20 +21,26 @@ const (
 type renderOpts struct {
 	manifest options.ManifestOptions
 	generic  options.GenericOptions
+
+	errOut io.Writer
 }
 
 // NewRenderCommand creates a render command.
-func NewRenderCommand() *cobra.Command {
-	renderOpts := &renderOpts{}
+func NewRenderCommand(errOut io.Writer) *cobra.Command {
+	renderOpts := &renderOpts{errOut: errOut}
 	cmd := &cobra.Command{
 		Use:   "render",
 		Short: "Render kubernetes controller manager bootstrap manifests, secrets and configMaps",
 		Run: func(cmd *cobra.Command, args []string) {
-			if err := renderOpts.Validate(); err != nil {
-				glog.Fatal(err)
+			var err error
+			defer func() {
+				renderOpts.handleError(cmd, err)
```

Comment:
clever

Reply:
don't see other option to shim into cobra command ;-)